### DISCUSSION TITLE
After upgrading to new software, cannot home or control laser

### DISCUSSION THREAD

**Discussion by louboardguy** [arkypita/LaserGRBL#1492] [Q&A]
I just upgraded my firmware and now my Pro has a limit error on start up, and I cannot home or control movement or the laser.

**Comment by gmmanonymus111** [comment]
This does not sound like a LaserGRBL problem. It seems the update reverted your settings to GRBL defaults. I suggest contacting the manufacturer for the correct `$` GRBL settings for your machine.
